tests/publiccloud/check_registercloudguest.pm

```diff
@@ -65,6 +65,40 @@ sub run {
             #    $instance->ssh_assert_script_run(cmd => "systemctl is-enabled guestregister-lic-watcher.timer");
             #}
 
+            # Temporary transition check:
+            # Azure & GCE images are migrating from regionsrv-enabler-azure.timer to guestregister-lic-watcher.timer.
+            # During the transition, exactly one of these timers must be active (running), but not both.
+            # See poo#186104 and https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/22572 where the check was disabled.
+            # TODO: Remove this block and uncomment block above once all Azure/GCE images have completed the migration to guestregister-lic-watcher.timer.
```

Comment:
Don't add TODOs. Better leave it out and keep it in tickets or in your local git repository, either in `git stash` or a temporary "WIP"-commit

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details